client/service/src/config.rs

```diff
@@ -111,6 +111,8 @@ pub struct Configuration {
 	pub base_path: Option<BasePath>,
 	/// Configuration of the output format that the informant uses.
 	pub informant_output_format: sc_informant::OutputFormat,
+	/// Refresh nodes allowlist after block imported TODO do i need this?
```

Comment:
I don't think you need this.

Reply:
Is it a concern to check the storage entry exists at every block import? It's probably not that heavy, but it would be good to learn more about this performance consideration.

Reply:
A block import happens every three to six seconds on average.
Looking up a storage entry should be in the order of nanoseconds or tens of nanoseconds.
Let's not worry about performances at this point.